container/docker/engine.py

```diff
@@ -85,7 +85,7 @@ def build_buildcontainer_image(self):
 
         :return: generator of strings
         """
-        assert_initialized(self.base_path)
+        assert_initialized(self.base_path, playbook=self.playbook)
```

Comment:
Of course, if you follow my suggestion to change the method signature of `assert_initialized`, this can be left alone.